src/Tizen.NUI/src/public/Theme/DefaultThemeCommon.cs

```diff
@@ -37,7 +37,7 @@ public Theme Create()
             // TextLabel style.
             theme.AddStyleWithoutClone("Tizen.NUI.BaseComponents.TextLabel", new TextLabelStyle()
             {
-                FontFamily = Tizen.NUI.FontFamily.UseSystemSetting,
+                FontFamily = "BreezeSans",
```

Comment:
Is it intended actions to change the default theme font family?

Reply:
Yes
Currently, the rpi image has only one System FontType, BreezeSans.

With this patch.. Settings API will be called only once, but that still introduces some overhead.

I think until multiple System FontTypes are available, it would be better to avoid the call entirely, even if it's just once.